Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SwiftTryCatch",
+    platforms: [
+        .iOS(.v12),
+        .tvOS(.v12),
+        .watchOS(.v5),
+        .macOS(.v10_14),
```

Comment:
Note that this platform support could likely be much earlier. Let me know what you'd like here. All I needed for my other project was this versioning.